service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/SøknadsbehandlingStatistikkMapper.kt

```diff
@@ -116,14 +116,11 @@ internal class SøknadsbehandlingStatistikkMapper(
     internal object FunksjonellTidMapper {
         fun map(søknadsbehandling: Søknadsbehandling) = when (søknadsbehandling) {
```

Comment:
Hva tenker dere rundt å sende inn vedtaket til statistikkmapperen istedenfor søknadsbehandlingen?

Reply:
Det er flere plasser vi heller ønsker å forholde oss til vedtak enn søknadsbehandling og/eller revurdering. Vi har med vilje tatt tak i så få av disse casene som mulig her, for å holde PR-en så spisset som mulig.